api/send-email.js

```
@@ -33,23 +33,46 @@ export default async function handler(req, res) {
+          ? conversationData.full_conversation
+              .split('\n\n')
+              .map(line => {
+                if (line.startsWith('Cliente:')) {
+                  return `<p style="margin: 10px 0;"><strong style="color: #2563eb;">Cliente:</strong> ${line.replace('Cliente:', '').trim()}</p>`;
+                } else if (line.startsWith('Asistente:')) {
+                  return `<p style="margin: 10px 0;"><strong style="color: #059669;">Asistente:</strong> ${line.replace('Asistente:', '').trim()}</p>`;
+                }
+                return line ? `<p style="margin: 10px 0;">${line}</p>` : '';
+              })
+              .join('')
+          : '<p style="color: #dc2626;">No se registró conversación</p>';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)